system/Commands/Database/MigrateRefresh.php

```diff
@@ -111,7 +111,7 @@ public function run(array $params)
 
 		if (ENVIRONMENT === 'production')
 		{
-			$force = array_key_exists('f', $params) || CLI::getOption('f');
+			$force = CLI::getOption('f');
```

Comment:
Why remove the `array_key_exists`? If this is removed, using `command('migrate:refresh -f')` will no longer be forced and will always have a prompt.

Reply:
before if, `$params` always replaced with ['b' => 0], so it never reached.

Reply:
except, before if, `$params` is going to be changed to : 

```php
$params['b'] = 0;
```

Reply:
Anyway, database commands aren't tested yet. So perhaps, the outright replacement of `$params` is a bug.

Reply:
yes, that's probably a possible bug, this change only ensure no unreacheable code removed.